recognition/scoring.py

```diff
@@ -44,14 +45,42 @@ def run(self):
 
 
 class ScliteJob(Job):
-    def __init__(self, ref, hyp, cer=False, sort_files=False, additional_args=None):
+    """
+    Run the Sclite scorer from the SCTK toolkit
+
+    Outputs:
+        - out_report_dir: contains the report files with detailed scoring information
+        - out_*: the job also outputs many variables, please look in the init code for a list
+    """
+
+    def __init__(
+        self,
+        ref: tk.Path,
+        hyp: tk.Path,
+        cer: bool = False,
+        sort_files: bool = False,
+        additional_args: Optional[List[str]] = None,
+        sctk_binary_path: Optional[tk.Path] = None,
+        hash_binary_path: bool = False,
```

Comment:
I don't like to introduce options to hash other parameters or not. Also I dislike that it is off by default.

I suggest instead to have 
```
__sis_hash_exclude__ = {"sctk_binary_path": None}
```
or equivalent implementation in hash function and then if None is given, we have the old behavior. and otherwise we use it always for the hash (similar to explicit returnn or rasr paths)

Reply:
Got it